File: c++/Input.cpp
#include <iostream>
using namespace std;
int main(){
    double  gajiKotor, potongan,pajak , gajiBersih, pengalaman ;
    cout << "Masukan Gaji Kotor anda  "; 
    cin  >> gajiKotor;
    cout << "Masukan Berapa tahun pengalaman anda bekerja anda  "; 
    cin >> pengalaman;
    pajak = 0.10;
    potongan = pajak*gajiKotor;
    gajiBersih = (gajiKotor-potongan) * pengalaman;
    cout<< "Gaji Bersihnya adalah Rp. = ";
    cout<< gajiBersih ;

} 

File: Java/input.java
import java.util.Scanner;

public class input {
    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        System.out.println("===== Masukkan Data Gaji =====");

        System.out.print("Masukan jumlah gaji kotor anda  :");
        int gajiKotor = input.nextInt();
 
        double potongan = 0.10 * gajiKotor;
        double gajiBersih  =( gajiKotor-potongan);

        System.out.print("Toal Gaji Bersih yang Dimiliki oleh Karyawan Adalah =" + gajiBersih );

    }
}